CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 ### Updates
 ### Bug Fixes
 
+## 2023-01-12
+### Bug Fixes
+- Fix img shortcode not rendering in the PDF document @Zalaras
```

Comment:
Nit: Might be nice to add the ticket URL here. So it makes Jira releases a bit easier :)